Dockerfile

```diff
@@ -1,20 +1,10 @@
-FROM python:3
+FROM python:3-alpine
 
-ENV TERRAFORM_VERSION=0.9.2
+ENV TERRAFORM_VERSION=0.9.4
```

Comment:
Would be good to be sure we are ready to upgrade, since once the new version writes the state file the old version refuses to work (as I found out yesterday when I was checking something against the new version). Would be happy to upgrade though.